### ISSUE TITLE
Wither Spawner Placement

### ISSUE THREAD

**Issue by ripSquid** [kami-blue/client#704]
**Describe the solution you'd like**
You can make a wither by placing them laying down. Add that placement "style" (Screenshot provided)

**Additional context**

![image](https://user-images.githubusercontent.com/44409107/79336592-4caa4980-7f24-11ea-9bcf-cde8644fc8f9.png)
Like that but without iron blocks.

**Comment by dominikaaaa** [comment]
Why would this be useful?

**Comment by ripSquid** [comment]
not really. maybe to bypass or something. Maybe faster placement of soulsand

**Comment by dominikaaaa** [comment]
Do withers even work when placed sideways?

**Comment by Vonr** [comment]
bruh. yes.
